doc/supervised.rst

```diff
@@ -152,7 +152,9 @@ neighbors (with same labels) of :math:`\mathbf{x}_{i}`, :math:`y_{ij}=0`
 indicates :math:`\mathbf{x}_{i}, \mathbf{x}_{j}` belong to different classes, 
 :math:`[\cdot]_+=\max(0, \cdot)` is the Hinge loss.
 
-.. topic:: Example Code:
+.. rubric:: Example Code
+
+A basic usage of this model with the Iris dataset from scikit-learn.
```

Comment:
With the use of `rubric` instead of `topic`, would we still get a warning if we did not have this description sentence? If not, we could probably remove it in most cases to make things more concise, as it is often clear from the code what it is doing (here, using LMNN on iris). We can of course keep a description for examples where the code is not self-explanatory or we want to emphasize more what it is doing.

Reply:
Just checked, we can remove the body of `.. rubirc::` and don't get any warning, so I removed all simple comments from almost everywhere except in `weakly_supervised.rst` line 67 and 100 to make a little bit a emphasis.